src/Microsoft.Health.Fhir.Api/Controllers/ExportController.cs

```diff
@@ -74,16 +75,18 @@ public class ExportController : Controller
 
         [HttpGet]
         [Route(KnownRoutes.Export)]
-        [ValidateExportHeadersFilter]
+        [ServiceFilter(typeof(ValidateExportRequestFilterAttribute))]
         [AuditEventType(AuditEventSubType.Export)]
-        public async Task<IActionResult> Export()
+        public async Task<IActionResult> Export(
+            [FromQuery(Name = KnownQueryParameterNames.DestinationType)] string destinationType,
+            [FromQuery(Name = KnownQueryParameterNames.DestinationConnectionSettings)] string destinationConnectionString)
         {
             if (!_exportConfig.Enabled)
             {
                 throw new RequestNotValidException(string.Format(Resources.UnsupportedOperation, OperationsConstants.Export));
             }
 
-            CreateExportResponse response = await _mediator.ExportAsync(_fhirRequestContextAccessor.FhirRequestContext.Uri);
+            CreateExportResponse response = await _mediator.ExportAsync(_fhirRequestContextAccessor.FhirRequestContext.Uri, destinationType, destinationConnectionString);
```

Comment:
this could just be passed in as DestinationInfo?

Reply:
Wanted to keep `DestinationInfo` usage within `Fhir.Core` and avoid using it in `Fhir.Api`. I don't think it makes a huge difference either way.